<launch>
  <node pkg="carf" type="main.py" name="carf" output="screen"/>
   <arg name="user_id" default="0000" />
      <param name="/user_id" value="$(arg user_id)" />
   <arg name="objective" default="ascending" />
      <param name="/objective" value="$(arg objective)" /> 

  <!-- Final tokens locations -->
  <rosparam param="solution_ascending">
    ['111', '158', '183', '192', '253']
  </rosparam>
 
  <rosparam param="tokens_number">
    [ '111', '158',  '183','192', '253','328', '376', '394', '449','463', '475', '486','517', '744',  '819']
  </rosparam>
 
<rosparam param="rfid_id">
   3581181951 3306978559 894864895 2247134207 1439793151 3315042303 1972340735 2768466943 2509992703 2777051135 4117136895 3852242175 88436991 2504166399 356806911
</rosparam>


</launch>
